gui_classes/DataContainer.py

```diff
@@ -0,0 +1,15 @@
+from sss.dataclasses import ExtractionType, AudioWave
+from dataclasses import dataclass
+
+
+@dataclass
+class DataContainer:
+    method: str = None
+    extraction_type: ExtractionType = None
+    input_track_name: str = None
+
+    audio_wave: AudioWave = None
+    sr: int = None
+
+    evaluation_references = {'vocals': None, 'drums': None, 'bass': None, 'other': None}
+    evaluation_results = {'vocals': None, 'drums': None, 'bass': None, 'other': None}
```

Comment:
Obawiam sie, e te wartosci inicjalizowane na None moga latwo powodowac wyjatki. Dosyc fajnie wylapuje to Pylance w VSC. Radze sprawdzic w szczegolnosci `GuiMain.py` oraz `GuiSave.py`, poniewaz zawieraja duzo warningow odnosnie mozliwego niewlasciwego uzycia wartosci None. I chyba już mi się nawet program na takim typie błędu wywalił, ale nie sprawdzałem dokładnie źródła problemu.

Oczywiscie najslodszym i najszybszym paliatywem bedzie zostawienie tego w pixdziec i zapomnienie o bledach, jesli podstawowy scenariusz przechodzi. I tak nikt nie bedzie tej aplikacji mocniej testowal.